src/modm/driver/can/mcp2515_impl.hpp

```diff
@@ -16,15 +16,17 @@
 #ifndef MODM_MCP2515_HPP
 	#error	"Don't include this file directly, use 'mcp2515.hpp' instead!"
 #endif
-
 #include "mcp2515_bit_timings.hpp"
 #include "mcp2515_definitions.hpp"
+#include <modm/architecture/driver/atomic/queue.hpp>
 #include <modm/architecture/interface/assert.hpp>
 
-
 #undef	MODM_LOG_LEVEL
 #define	MODM_LOG_LEVEL modm::log::DISABLED
 
+static modm::atomic::Queue<modm::can::Message, 32> txQueue;
+static modm::atomic::Queue<modm::can::Message, 32> rxQueue;
```

Comment:
Don't we have to define these inside of the driver class as class members? If we have more than one MCP2515 instance both will use the same buffer.

Reply:
Oh yeah I forgot that ... Nearly all of the stuff is templated via the class so it's fine ... But here I forgot do so something, I'll fix that tomorrow